src/js/components/modals/AppModalComponent.jsx

```diff
@@ -98,7 +99,12 @@ var AppModalComponent = React.createClass({
     // All status below 300 are actually not an error
     if (status < 300) {
       this.onCreateApp();
-
+    } else if (status === 409) {
+      this.setState({
+        responseErrorMessages: {general: `Failed to change and deploy the new
```

Comment:
I would write it this way:

`Error: App is locked by one or more deployments. Pressing the button again will forcefully change and deploy the new configuration.`